angularSTEP/src/app/app.module.ts

```diff
@@ -34,6 +34,8 @@ import {UploadRecipeComponent} from './upload-recipe/upload-recipe.component';
 import {ViewProfilesComponent} from './view-profiles/view-profiles.component'; 
 import {UserPageComponent} from './user-page/user-page.component'; 
 import {environment} from '../environments/environment';
+import { MapComponent } from './map/map.component';
+import { GoogleMapsModule } from '@angular/google-maps';
 import {MatAutocompleteModule} from '@angular/material/autocomplete';
```

Comment:
Can we please make a final decision on this spacing? I know it seems to be all I comment on but I think all of us end up having inconsistencies due to not being clear on what we've chosen. I know in a previous PR you said that gts fix removes the spaces, so I thought that we were going with that, but can we please all just be on the same page.

Reply:
Okay. I tried to go back through each file and fix them to contain no spaces. Thank you for checking. The problem of merging keeps rising up so even though I try to fix it, when merging it gets changed again. I will try to stay on top of it.

Reply:
I don't mean to make it sound like I'm putting blame on you or any one person. It's just that every time I look at a PR it seems to be that we all have something different, so we should all probably come to an agreement so we can all be aware and keep the merges from reverting things.